todo/views.py

```diff
@@ -0,0 +1,75 @@
+from django.shortcuts import get_object_or_404, redirect
+from django.urls import reverse_lazy
+from django.views import generic, View
+
+from todo.forms import TagForm, TaskForm
+from todo.models import Task, Tag
+
+
+class TaskListView(generic.ListView):
+    model = Task
+    context_object_name = "task_list"
+    template_name = "todo/task_list.html"
+    paginate_by = 3
```

Comment:
As we can see `TaskListView` and `TagListView` use the same value of pagination. It could be extracted to another class called `PaginationMixin` and then inherited by both classes. It will simplify further changes.

Please read about mixins in Python